ai.finto.fi/static/js/annif.js

```diff
@@ -254,7 +323,9 @@ const mainApp = createApp({
       })
       .then(data => {
         this.projects = data.projects
-        this.selected_project = this.projects[0].project_id
+        // Assume vocabulary id is a prefix of project id
```

Comment:
Ideally, the vocab ID would be returned by the Annif REST API, but it doesn't expose it currently. I opened an [issue](https://github.com/NatLibFi/Annif/issues/837) about it on the Annif side.